hasher-matcher-actioner/hmalib/common/configs/actioner.py

```diff
@@ -40,6 +40,13 @@ def perform_action(self, match_message: MatchMessage) -> None:
         raise NotImplementedError
 
 
+# at match time, these strings are replaced by the result of running the associated
+# function on the match message
+WEBHOOK_ACTION_PERFORMER_REPLACEMENTS: t.Dict[str, t.Callable[[MatchMessage], str]] = {
```

Comment:
nit/ignorable: this could be inline now that we are down to one value. Especially imo as for changes beyond this I'd actually like to see the replacement be part of the action config, e.g. provide specifics of how to parse the values added to 'additional_fields' etc.

Reply:
I like this as separate so its clear where to make future additions which I think we may want